engine/src/main/scala/cromwell/engine/backend/dummy/DummyBackendJobExecutionActor.scala

```diff
@@ -0,0 +1,46 @@
+package cromwell.engine.backend.dummy
+
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionSucceededResponse, BackendJobExecutionResponse}
+import cromwell.backend.BackendLifecycleActor.{BackendJobExecutionAbortSucceededResponse, JobAbortResponse}
+import cromwell.backend._
+import cromwell.core.CallOutput
+import wdl4s.types._
+import wdl4s.values._
+import wdl4s.{TaskOutput, Call}
+
+import scala.concurrent.Future
+
+case class DummyBackendJobExecutionActor(workflowDescriptor: BackendWorkflowDescriptor, configurationDescriptor: BackendConfigurationDescriptor, calls: Seq[Call]) extends BackendJobExecutionActor {
```

Comment:
This is just for a quick proof of concept check? Or can we move this to the tests?